plasma_framework/migrations/2_deploy_plasma_framework.js

```diff
@@ -22,5 +22,5 @@ module.exports = async (
     const plasmaFramework = await PlasmaFramework.deployed();
     await plasmaFramework.activateChildChain({ from: authorityAddress });
     const sha = childProcess.execSync('git rev-parse HEAD').toString().trim().substring(0, 7);
-    await plasmaFramework.setVersion(`${pck.version}+${sha}`), { from: maintainerAddress });
+    await plasmaFramework.setVersion('${pck.version}+${sha}'), { from: maintainerAddress });
```

Comment:
lol, this commit is wrong. you need to use ` instead of ' to be able to put variable in js.

Reply:
lol, I thought Kevin made a typo :D